sleap/nn/inference.py

```diff
@@ -228,6 +280,16 @@ def from_model_paths(
                 resize_input_layer=resize_input_layer,
             )
 
+        elif "thunder" == model_types or "lightning" == model_types:
```

Comment:
Taking a closer look at this, I don't think this will work... We may need to add a check (for a movenet model) at the top of this function before we start trying to load model configs.
https://github.com/talmolab/sleap/blob/ad610fb506023cd99e9dfcf394d970f528bf276e/sleap/nn/inference.py#L205-L211

Reply:
I think the way to go would be to add another keyword argument to the `Predictor.from_model_paths` called `model_name` with default to `None`. We would check if the `model_name` is specified at the top of `Predictor.from_model_paths` and try to load the model from the `model_name` first.

Reply:
This expands our `Predictor.from_model_paths` function to include any pre-trained models (which we don't plan on retraining).